<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
    <param name="click_path"    value="$(find automatic-camera-pointcloud-calibration)/data/corner_lidar.txt" />  <!-- output file -->
    <param name="selected_number"    value="23" />  <!-- number of points to be selected, please accord with the camera corner points -->
    
    <node pkg="automatic-camera-pointcloud-calibration" name="lidar_record" type="lidar_record" output="screen"></node>


</launch>

